internal/service/wallet.go

```diff
@@ -359,6 +365,9 @@ func (s *walletService) TerminateWallet(ctx context.Context, walletID string) er
 			return err
 		}
 
+		// Publish webhook event
+		s.publishWebhookEvent(ctx, types.WebhookEventWalletTerminated, walletID)
```

Comment:
move this outside the `s.DB.WithTx` 
can do like
```
err = s.DB.WithTx .....
if err != nil {
return err
}
publish webhook
return nil
```

We should never publish webhook inside a DB transaction